views/layouts/form.blade.php

```diff
@@ -110,6 +110,27 @@
     <a17-modal class="modal--browser" ref="browserWide" mode="wide" :force-close="true">
         <a17-browser></a17-browser>
     </a17-modal>
+    @foreach ($browsersWithCreate as $browser)
+        @php
+          $fieldsInModal = $moduleName != $browser['moduleName'];
+          $titleIsTranslatable = count($browser['languages']) > 1;
```

Comment:
I'm still testing this but I believe this will not work when translations in use, but only one language is configured. The `title` field is not saved correctly in this context.

This is a bit of an edge case but can happen on projects where translations arrive later in the process.